### DISCUSSION TITLE
How to order by closest to position (longLat) in Prisma? (Nodejs)

### DISCUSSION THREAD

**Discussion by abdelkadersebihi** [prisma/prisma#16073] [Q&A] [answered]
```
const stores = await prisma.store.findMany({
          where: {
            isActive: true,
          },
          take: 10,
          orderBy: {
             //TODO: order stores by closes to req.query.long and req.query.lat
          },
});
```

**Comment by nurul3101** [comment, marked as answer]
Hey @aymensbh 👋 

You are looking for Geospatial queries, Prisma doesn't have official support for PostGIS/GIS, but we do have a Feature Request for it: #2789, we are currently working on supporting it, you can expect support for it in future releases. It would help if you can leave a 👍 on the Feature Request.

For now, you would need to use [Raw Queries](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access) as a workaround. Do let me know in case you have any other queries.

**Reply by abdelkadersebihi** [reply]
Okay thanks

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
